frontend/css/rodi_css.css

```diff
@@ -359,6 +359,24 @@ h1 + .lead {
     font-weight: 500;
 }
 
+
+/*
+==============================================================================
+    truncate lines
+==============================================================================
+*/
+
+
+
+.trunkate {
```

Comment:
There is already `.single-line` CSS class which does exactly this. And which works also for Firefox.

This snippet works only for Chrome.

Reply:
True, yes